spec/_attachments/ic.did

```diff
@@ -115,6 +115,13 @@ type send_transaction_request = record {
 
 type millisatoshi_per_byte = nat64;
 
+type snapshot_id = nat;
+type snapshot = record {
+  id: snapshot_id;
+  taken_at_timestamp: nat64;
```

Comment:
This field seems never used - is it supposed to be read by developers when listing snapshots?

Reply:
although they could also note down the time when they took the snapshot themselves

Reply:
> This field seems never used - is it supposed to be read by developers when listing snapshots? 
Yes, that is the case. This field, I think, will be especially useful when we allow more snapshots per canister. I will leave this field as unless there is a strong preference to remove it.